jpa/hibernate7/pom.xml

```diff
@@ -90,6 +90,12 @@
             <artifactId>hibernate-scan-jandex</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.github.ben-manes.caffeine</groupId>
```

Comment:
I don't recall if WildFly has any other dependencies that depend on a personal github repo like https://github.com/ben-manes/caffeine.  I'm not sure if this is a concern for WildFly but pointing it out so it won't be missed by the reviewers.